Centaurus.Domain/Transactions/StellarPaymentsProvider.cs

```diff
@@ -0,0 +1,239 @@
+﻿using Centaurus.Models;
+using Centaurus.Stellar;
+using Centaurus.Stellar.Models;
+using NLog;
+using stellar_dotnet_sdk;
+using stellar_dotnet_sdk.responses;
+using stellar_dotnet_sdk.xdr;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Timers;
+
+namespace Centaurus.Domain
+{
+    public class StellarPaymentsProvider : PaymentsProviderBase<KeyPair, KeyPair, long>
```

Comment:
I don't like generics here. A provider should hide implementation details if possible.